bin/set-vars.sh

```diff
@@ -1,14 +1,10 @@
 #!/usr/bin/env bash
 BRANCH=`git branch --show-current`
 COMMIT=`git rev-parse --short HEAD`
-if [  -z "$GITHUB_REF" ]; then
-    TAQ_VERSION="dev-$BRANCH"
+if [  -z "$GITHUB_REF_NAME" ]; then
+    TAQ_VERSION="dev-${BRANCH//\//-}"
 else
-    TAQ_VERSION="${GITHUB_REF##*/}"
+    TAQ_VERSION="${GITHUB_REF_NAME//\//-}"
```

Comment:
We can remove the `if ` condition by using `BRANCH=git rev-parse --abbrev-ref HEAD` which will work on both GH runners and locally